docs/src/modules/components/Demo.js

```diff
@@ -250,13 +250,17 @@ export default function Demo(props) {
         </NoSsr>
       )}
       <Collapse in={openDemoSource} unmountOnExit>
-        <div>
-          <Code
-            id={demoSourceId}
-            code={showPreview && !codeOpen ? demo.jsxPreview : demoData.raw}
-            language={demoData.sourceLanguage}
-          />
-        </div>
+        <Code
+          id={demoSourceId}
+          code={showPreview && !codeOpen ? demo.jsxPreview : demoData.raw}
+          language={demoData.sourceLanguage}
+          enableCodeCopy
+          analytics={{
```

Comment:
Would it be clearer with this name? It seems that it's a straight prop drilling pattern

```suggestion
          copyButtonProps={{
```